### ISSUE TITLE
test step for Mmg v5.6.0 fails because ci_tests directory is missing

### ISSUE THREAD

**Issue by boegel** [easybuilders/easybuild-easyconfigs#15662]
Pretty much all tests fail for me with errors like:

```
222/245 Test #224: mmg_LeakCheck_args1_mmg3d_O3 ......................***Failed    0.14 sec
  ** /tmp/easybuild_build/Mmg/5.6.0/gompi-2021b/mmg-5.6.0/ci_tests/mmg/LeakCheck_args1/d  NOT FOUND.
  ** UNABLE TO OPEN INPUT FILE.
  -- MMG3D, Release 5.6.0 (Nov. 05, 2021) 
     Copyright (c) Bdx INP/CNRS/Inria/UPMC, 2004-
     Jun 10 2022 21:39:27

  -- INPUT DATA

   ELAPSED TIME  0.000s
```

(see also test report at https://github.com/easybuilders/easybuild-easyconfigs/pull/15622#issuecomment-1152675085)

I'm not sure how this wasn't a problem for others in #15622... Any ideas here @Micket, @SebastianAchilles?

**Comment by branfosj** [comment]
The tests also fail for me.

Checking the CMake configure stage, I see:
```
-- Offline mode: requires already downloaded test cases
```

That gets me to `cmake/modules/LoadCiTests.cmake`, which sets offline status on the result of:
```
ping www.mmgtools.org -c 2
```

That ping works from my laptop but not from from our HPC.

**Comment by branfosj** [comment]
The CMake that downloads the tests does this inside the unpacked source directory:

```bash
mkdir ci_tests
cd ci_tests
for x in '' 2d 3d s; do
  wget https://static.bordeaux.inria.fr/mmg/mmg${x}.tgz
  wget https://static.bordeaux.inria.fr/mmg/mmg${x}.version
  tar xf mmg${x}.tgz
  rm mmg${x}.tgz
done
```

**Comment by Micket** [comment]
Well that sucks. They filenames aren't versioned, so, presumably, this is going to be break in the future even if we go through the hassle of pre-downloading these as part of the sources.

I see a hope though
```
  IF ( NOT ONLY_VERY_SHORT_TESTS )
    ##-------------------------------------------------------------------##
    ##------- Get the continuous integration tests ----------------------##
    ##-------------------------------------------------------------------##
    INCLUDE(cmake/modules/LoadCiTests.cmake)
  ENDIF ( )
```
So, we can limit it to only the very shorts tests then i guess and that'll be something at least...

But, they are really very very short.
```
1/3 Test #1: mmg2d_very_short .................   Passed    0.01 sec
2/3 Test #2: mmgs_very_short ..................   Passed    0.20 sec
3/3 Test #3: mmg3d_very_short .................   Passed    1.00 sec
```

but, there are many tests for the library itself, the cmake files have sprinkled in some binaries whose source code is actually inside the ci-tests tarballs.. ugh, but they are straight foward to patch out. We should really run those tests also, which we aren't even now doing.

**Comment by Micket** [comment]
Fixed in #15691
